tools/depcheck/pkg/cmd/pkg_test.go

```diff
@@ -1,4 +1,4 @@
-package trace
+package cmd
```

Comment:
you're picking up on bad patterns in the main code.  package "cmd" is a bad idea, not a good one.  These are all related to the graph, just not methods on it.  You could co-locate